### ISSUE TITLE
Vehicle and kit Suggestion

### ISSUE THREAD

**Issue by Nama023** [UncreatedStaff/UncreatedWarfare#102]
**Multipurpose jet**
Remove the 2nd seat (AIM 9) and make it into pilot seat, so pilot can use cannon and air to air missile without changing seat. It's actually possible (https://steamcommunity.com/sharedfiles/filedetails/?id=2772030264) by pressing unlock/lock button to switch the gun.

**HAT Kit**
I wish the SMG on HAT kit had a sight/scope, the custom kit for hat is really op they can PvP and kill vehicle at same time.

**MBT**
Able to carry more shells + bring back ability for driver to switch into MG seat

**Comment by UsernameIsUnavailible** [comment]
Multiple guns for one seat
-> No.
HAT Kit
-> ¯\_(ツ)_/¯
MBT
-> Would make it too OP if we allow it to carry more shells, Currently you can already carry roughly 10-15shells per person, times three is 30-45 shells.. Driver shouldn't change seats to react quickly in combat situations.

**Comment by Nama023** [comment]
1. Why not ? There's literally fighter jet
3. For shells, yes it's make it op but not too OP. Driver shouldn't change but how come the gunner seat able to switch into f3

**Comment by UsernameIsUnavailible** [comment]
Because adding multiple guns for a seat for jets would make them even more OP. Jets have been tweaked for S3 and probably will not change, Using multiple arments via the lock button or plugin button is going to become messy and unfriendly to beginners or players that do not know the button, not to mention the string of code for the ammo, reloading, using it....

Gunner can change to F3 since... nothing has been coded to stop him from doing it??

**Comment by Nama023** [comment]
It's not that OP otherwise it's really cool feature, anyways the co pilot can shoot air to air missile so nothing different it's just make the thing abit complicated since it has 4 seats now. But back again it's depend on 420 if he want to change it

**Comment by HamzaXSniper** [comment]
They added a machinegun to jets and moved a2a missiles to f2 for a reason, mixing them together removes the entire purpose of the separation and the solo fighter jet.

**Comment by Nama023** [comment]
Not really, CO pilot can bring more a2a missiles. So that multirole jet is even more powerful fighter jet. They can shoot a2a and cannon in sametime.

**Comment by UsernameIsUnavailible** [comment]
> Not really, CO pilot can bring more a2a missiles. So that multirole jet is even more powerful fighter jet. They can shoot a2a and cannon in sametime.

are you stupid, both will use same kit

**Comment by Nama023** [comment]
i know michael, my point is to limit air to air missile for multirole. So that jet can't shoot more than 4 a2a since it has cannon.

**Comment by UsernameIsUnavailible** [comment]
good luck hitting anything with an autocannon?

**Comment by Nama023** [comment]
It's not that hard to hit with autocannon , the a2a for multirole is supposed to be limited anyways. Then it's no point of fighter jet :/

**Comment by ToastyTimeZ** [comment]
It'd be better idea if you had suggested it better, buff a2a missiles, only allow 2 per clip and make ot require 2 hits with them, make autocannon do only low cal dmg and .48 dmg per fragmentation, this way it isn't an entire tank buster

**Comment by ToastyTimeZ** [comment]
Of course it'd be still not enough for getting an autocannon added, because the a2a missiles can still be used on vehicles and do shit tons of dmg, it takes I think 3 or 4 direct a2a missile hits to kill a tank

**Comment by benvalkin** [comment]
I'll need to know a bit more about that proposed implementation works before I can have an opinion on that idea. Also, I'd prefer not to use too many plugin keys & rather keep core features (as opposed to making everything highly dependent on plugins).

 I don't really care about HAT kit scopes but I've been thinking of reducing the shell item size to 3x1 down from 4x1 to make MBTs more fun

**Comment by benvalkin** [comment]
Also @Nama023 in future could you please post only one idea in each issue? It's fine even if you have to post 3 separate issues, but if you put 3 different ideas in one issue then we can't set the status or close the issue correctly

**Comment by Nama023** [comment]
> Also @Nama023 in future could you please post only one idea in each issue? It's fine even if you have to post 3 separate issues, but if you put 3 different ideas in one issue then we can't set the status or close the issue correctly

Alright
